From ca26ed72ef41c5202734c934d00468b3c59b92ad Mon Sep 17 00:00:00 2001
From: Neena Busireddy <neenareddy.busireddy@nxp.com>
Date: Thu, 13 Sep 2018 17:53:31 +0000
Subject: [PATCH] CMakeLists.txt: Fix compilation issues with layersVT

Not able to find the c++ headers if CMAKE_CXX_FLAGS is set to -std=c++11

Signed-off-by: Neena Busireddy <neenareddy.busireddy@nxp.com>
---
 layersvt/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/layersvt/CMakeLists.txt b/layersvt/CMakeLists.txt
index 3c60550..e37e7ae 100644
--- a/layersvt/CMakeLists.txt
+++ b/layersvt/CMakeLists.txt
@@ -151,7 +151,6 @@ if (WIN32)
         set (CMAKE_C_FLAGS_DEBUG     "${CMAKE_C_FLAGS_DEBUG} -D_CRT_SECURE_NO_WARNINGS")
     endif()
 else()
-    set (CMAKE_CXX_FLAGS "-std=c++11")
     set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wpointer-arith")
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wpointer-arith")
 endif()
-- 
1.9.1

